### DISCUSSION TITLE
Unable to connect to resource: Plex Server.

### DISCUSSION THREAD

**Discussion by dariokan82** [Taxel/PlexTraktSync#1672] [Support] [answered]
Hello folks,

I'm getting crazy with this issue. I deploy the container and run it, it authenticates in Plex at first login without a issue, and then sees my server, but when I try to connect, it gives me the error as per subject, adding:

`ERROR    https://192-168-4-57.7ee5205cf46ccbf6cebad1d2aacd0.plex.direct:32400:
         HTTPSConnectionPool(host='192-168-4-57.7ee5205cf46ccbf6cebad1d2aacd0.plex.direct', port=32400): Max retries
         exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at
         0x7f97608df390>, 'Connection to 192-168-4-57.7ee5205cf46ccbf6cebad1d2aacd0.plex.direct timed out. (connect
         timeout=30)'))`

nothing I can do seems to work. Any help is appreciated!

**Comment by simonc56** [comment]
Give full debug logs (not only one line).
And same thing for Plex server.

**Comment by glensc** [comment]
This is typically networking issue, not problem of PlexTraktSync.

Try checking connection with system tools. ping, telnet, netcat

**Comment by glensc** [comment]
The login process may save multiple urls, see your servers.yml and remove unreachable URLs from it.

And in login screen, what server adress was connecting successfully? try to ensure it's in servers.yml as first url

**Reply by dariokan82** [reply, marked as answer]
Thanks. I managed to recreate the container and bind it to the host network, and it worked! :)
